doc/source/cross-language.rst

```diff
@@ -0,0 +1,293 @@
+Cross-language programming
+==========================
+
+This page will show you how to use Ray's cross-language programming feature.
+
+Python calling Java
+-------------------
+
+Suppose we have a Java static method and a Java class as follows:
+
+.. code-block:: java
+
+  package io.ray.demo;
+
+  public class Math {
+
+    public static int add(int a, int b) {
+      return a + b;
+    }
+  }
+
+.. code-block:: java
+
+  package io.ray.demo;
+
+  // A regular Java class.
+  public class Counter {
+
+    private int value = 0;
+
+    public int increment() {
+      this.value += 1;
+      return this.value;
+    }
+  }
+
+Then, in Python, we can call the above Java remote function, or create an actor
+from the above Java class.
+
+.. code-block:: python
+
+  import ray
+
+  ray.init(_include_java=True, _load_code_from_local=True)
```

Comment:
`_include_java` will be removed soon, let's just remove it here as well

Reply:
Fixed.